### ISSUE TITLE
jlink not working  (javafx:jlink) 

### ISSUE THREAD

**Issue by msgilligan** [SupernautApp/supernaut-fx-sample-maven#1]
When running the `mvn javafx:jlink` goal, the following error occurs:

```
Error: automatic module cannot be used with jlink: io.micronaut.inject from file:///Users/sean/.m2/repository/io/micronaut/micronaut-inject/2.3.1/micronaut-inject-2.3.1.jar
```

Some processing of the automatic modules similar to what we're doing with Badass JLink Plugin in the sample Gradle build, needs to occur for `jlink` to work properly.

**Comment by msgilligan** [comment]
The first thing to try is probably the [Maven Moditect](https://github.com/moditect/moditect) plugin.

@FDelporte do you think you might be able to give it a try?

**Comment by msgilligan** [comment]
PR #5 was merged to the `develop` branch and seems to build a runable `jlink` image - thanks @chirontt!

Note that the `javafx:jlink` goal still doesn't work, but a `jlink` image is built if you run `mvn verify`

I'm not sure what to do to generate `jpackage` image(s), but I suppose we can open a new Issue for that.

**Comment by msgilligan** [comment]
Fixed by PR #5 and README was updated by 905e7718e5934b1270d9227b63026fb6607e4f06
